crates/ruma-events/src/secret_storage/key.rs

```diff
@@ -59,7 +59,7 @@ fn is_default_bits(val: &UInt) -> bool {
 /// essentially represents `AesHmacSha2KeyDescription` in the
 /// [spec](https://spec.matrix.org/latest/client-server-api/#msecret_storagev1aes-hmac-sha2).
 #[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
-#[derive(Clone, Debug, Serialize, EventContent)]
+#[derive(Clone, Debug, Serialize, Deserialize, EventContent)]
```

Comment:
This is wrong, we need the event type when deserializing this event content to be able to fill the `key_id`, which is why it doesn't implement `Deserialize`.

Reply:
Indeed! Fixed in the latest version.